src/services/classes/regenerate_color.js

```diff
@@ -0,0 +1,25 @@
+function getRandomColor() {
+    return [
+        Math.floor(Math.random() * 256),
+        Math.floor(Math.random() * 256),
+        Math.floor(Math.random() * 160),
+    ];
+}
+
+async function reGenerateColor(context) {
+    const users = context.app.service('users');
+
+    const userColors = await users.find({
+        query: {
+            permissions: 'student'
+        }
+    });
+
+    userColors.forEach( element => {
+        if (element.color[2] > 160 && element.permissions == 'student') {
+            users.patch(element.username, { color: getRandomColor() });
+        };
+    });
```

Comment:
I think you can also destruct element like this to obtain all the parameters you need:

```suggestion
    userColors.forEach(({ color: [,, blue], permissions, username }) => {
        if (blue > 160 && permissions == 'student') {
            users.patch(username, { color: getRandomColor() });
        };
    });
```